index.bs

```diff
@@ -4648,71 +4659,151 @@ The <dfn export for=commands>browsingContext.setViewport</dfn> command modifies
    </dd>
 </dl>
 
-<div algorithm="remote end steps for browsingContext.setViewport">
+<div algorithm>
+To <dfn>set device pixel ratio override</dfn> given |navigable| and |device pixel ratio|:
 
-The [=remote end steps=] with |command parameters| are:
+1. If |device pixel ratio| is not null:
 
-1. Let |navigable id| be the value of the <code>context</code> field of |command
-   parameters|.
+   1. When the [=select an image source from a source set=] steps are run, act as if
```

Comment:
Should this say something about what it applies to e.g. "for [=documents=] loaded in |navigable|"?

Reply:
yeah, I guess it applies to the second step is well, so I've added before to work for both